--- a/node_modules/@bpmn-io/form-js-viewer/dist/index.es.js	2022-09-29 20:59:09.846120299 +0300
+++ b/node_modules/@bpmn-io/form-js-viewer/dist/index.es.js	2022-09-29 20:59:09.846120299 +0300
@@ -2166,9 +2166,10 @@
       id: prefixId(id, formId),
       label: label,
       required: required
-    }), jsx("input", {
+    }), jsx(!!(field && field.key && field.key.match(/text|comment|feedback|body/)) ? "textarea" : "input", {
       class: "fjs-input",
       disabled: disabled,
+      rows: !!(field && field.key && field.key.match(/text|comment|feedback|body/)) ? 10 : null,
       id: prefixId(id, formId),
       onInput: onChange,
       type: "text",
